circle.yml

```diff
@@ -24,6 +24,15 @@ checkout:
     - case $CIRCLE_NODE_INDEX in 0) echo "Skipping step." ;; *) ./bin/circle/merge-base ;; esac:
         parallel: true
 dependencies:
+  pre:
+    # update google chrome; Selenium requires that Chrome version must
+    # be >= 52.0.2743.0
+    - google-chrome --version
+    - wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+    - sudo sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
+    - sudo apt-get update
+    - sudo apt-get --only-upgrade install google-chrome-stable
+    - google-chrome --version
```

Comment:
Can/should this get wrapped into a helper script?  I've been wondering if we should have more of our steps bundled up to keep our circle.yml more concise.  Also, can we usefully verify that the chrome version is good here?